components/pushbullet/actions/send-push/send-push.mjs

```diff
@@ -0,0 +1,149 @@
+import pushbullet from "../../pushbullet.app.mjs";
+import constants from "../common/constants.mjs";
+
+export default {
+  name: "Send push (Note, Link and File)",
+  version: "0.0.1",
+  key: "pushbullet-send-push",
+  description: "Send a push to a device or another person. [See docs here](https://docs.pushbullet.com/#create-push)",
+  type: "action",
+  props: {
+    pushbullet,
+    device: {
+      propDefinition: [
+        pushbullet,
+        "device",
+      ],
+    },
+    email: {
+      label: "Email",
+      description: "Send the push to this email address. If that email address is associated with a Pushbullet user, we will send it directly to that user, otherwise we will fallback to sending an email to the email address (this will also happen if a user exists but has no devices registered).",
+      type: "string",
+      optional: true,
+    },
+    channel_tag: {
+      label: "Channel Tag ",
+      description: "Send the push to all subscribers to your channel that has this tag.",
+      type: "string",
+      optional: true,
+    },
+    type: {
+      label: "Push Type",
+      description: "The type of push you want to send",
+      type: "string",
+      options: constants.PUSH_TYPES,
+      reloadProps: true,
+    },
+  },
+  async additionalProps() {
+    const props = {};
```

Comment:
I'd say to use a `switch/case`  instead of these 3 `if`
Also, in my view, it would be better if you directly return the `additionalProps` inside each `case` of your `switch`